srcpkgs/zellij/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'zellij'
+pkgname=zellij
+version=0.17.0
+revision=1
+# Dependency crate `wasmer-vm` only supports the following archs: x86_64, arm64, x86
+# https://docs.wasmer.io/ecosystem/wasmer/wasmer-features#compiler-support-by-chipset
+archs="x86_64* aarch64* i686"
+build_style=cargo
+build_helper=qemu
+hostmakedepends="pkg-config"
+makedepends="openssl-devel"
+short_desc="Terminal workspace with batteries included"
+maintainer="Kartik Singh <kartik.ynwa@gmail.com>"
+license="MIT"
+homepage="https://github.com/zellij-org/zellij"
+distfiles="https://github.com/zellij-org/zellij/archive/refs/tags/v${version}.tar.gz"
+checksum=a1c283e6200199ad7bdd286a4dbba1e4906bac1fa1471ba6b45c22ef55c8808d
+
+post_build() {
+	STARSHIP="target/${RUST_TARGET}/release/zellij"
```

Comment:
You probably copied from starship, but i think it makes sense to change the variable's name :p

Reply:
Done